librz/il/definitions/bool.c

```diff
@@ -0,0 +1,69 @@
+// SPDX-FileCopyrightText: 2021 heersin <teablearcher@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_il/definitions/bool.h>
+
+/**
+ * Create a new RzILBool instance
+ * \param true_or_false bool, set bool as true or false
+ * \return bool RzILBool, pointer to bool value
+ */
+RZ_API RzILBool rz_il_new_bool(bool true_or_false) {
+	RzILBool ret = (RzILBool)malloc(sizeof(struct bool_t));
+	ret->b = true_or_false;
+	return ret;
+}
+
+/**
+ * result of a `AND` b
+ * \param a RzILBool, operand of `AND`
+ * \param b RzILBool, operand of `AND`
+ * \return bool RzILBool, pointer to the result
+ */
+RZ_API RzILBool rz_il_bool_and_(RzILBool a, RzILBool b) {
```

Comment:
Please don't forget about those `_`

Reply:
this `_` existed in original Core Theory, do we remove this suffix in our implementation ? @XVilka

Reply:
Yes, I explained why - in OCaml `and`, `or`, etc are a keywords, thus they called functions `and_`, `or_` etc. Purely a technical limitation. In our case it's unnecessary.